src/query/builder.js

```diff
@@ -853,6 +853,15 @@ assign(Builder.prototype, {
     return this;
   },
 
+  // Insert on conflict do nothing/update
+  onConflict(columns, updates) {
```

Comment:
I think this should also allow passing raw() so that people can also write their own code, what to do in case of conflict (sqlite has quite a different selection than nothin / update set https://sqlite.org/lang_conflict.html).